File: sqlInsert.py
import mysql.connector

# Open database connection
db = mysql.connector.connect(host="192.168.0.131", user="phiric21", passwd="richmeier", db="phiric21")

# prepare a cursor object using cursor() method
cur = db.cursor(dictionary=True)

name='Jerome'
age = 14
grade = 10

# create table
sql = "INSERT INTO students (name, age, gradeLevel) VALUES ('Jerome', 15,10)"
sql2 = "SELECT * from students"

cur.execute(sql)
cur.execute(sql2)

rows = cur.fetchall()
for row in rows:
	print("Name: "+str(row['name']) + " Age: " +str(row['age']) + " Grade Level: " +str(row['gradeLevel']))

# disconnect from server
cur.close()
db.close()

File: sqlInsert.php
<?php
$servername = "localhost";
$username = "phiric21";
$password = "password";
$dbname = "school";

// Create connection
$conn = new mysqli($servername, $username, $password, $dbname);
// Check connection
if ($conn->connect_error) {
    die("Connection failed: " . $conn->connect_error);
}

// write out the data being inserted into the table in values
$sql = "INSERT INTO students (name, age, gradeLevel) VALUES ('John', '22', '15')";

// when you connect to the server echo out New record created successfully
if ($conn->query($sql) === TRUE) {
    echo "New record created successfully";
} else {
    echo "Error: " . $sql . "<br>" . $conn->error;
}

$conn->close();
?>
